src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/summaries/HotResourceSummary.java

```diff
@@ -60,6 +60,16 @@
   private String metaData;
   private List<TopConsumerSummary> topConsumerSummaryList;
 
+  private UsageBucket usageBucket;
```

Comment:
HotResourceSummary is shared among various RCAs and is closed coupled with rpc / SQL persistor so we don't want to add extra variable unless it is necessary. Since we are emitting bucket information from cluster level RCA, can we define a new type of flowunit and add bucket object into the flowunit itself ?

Reply:
Moved into ClusterFlowUnit and the associated classes